### PR TITLE
Update tracing documentation

### PR DESCRIPTION
**Pull request by grampelberg**
This is paired with a PR in [emojivoto](https://github.com/BuoyantIO/emojivoto/pull/80).

### CODE REVIEW

File: run.linkerd.io/public/tracing/collector.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    protocol: TCP
+    targetPort: 55678
+  - name: zipkin
+    port: 9411
+    protocol: TCP
+    targetPort: 9411
+  selector:
+    component: oc-collector
+---
+apiVersion: extensions/v1beta1
```
Comment (ihcsim) [inline run.linkerd.io/public/tracing/collector.yml]: Better to use `apps/v1`, and then add `selector` to the `spec`:

```
spec:
  selector:
    matchLabels:
      app: opencensus
      component: oc-collector
```

File: run.linkerd.io/public/tracing/backend.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: tracing
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
```
Comment (ihcsim) [inline run.linkerd.io/public/tracing/backend.yml]: Better to use `apps/v1` here, and then add the `selector` to the `spec`:

```
spec:
  selector:
    matchLabels:
      app: jaeger
```

**Comment by ihcsim** [comment]
I assume this fixes https://github.com/linkerd/website/issues/575.

**Comment by alpeb** [comment]
Just a heads-up, one of the emojivoto links still points to Alex's fork:
https://github.com/linkerd/website/pull/582/files#diff-000fe5a1c5b8ad0f49a4ab6abfa7d72dR253

**Pull request merged**